src/app/features/template-export/template-export-file-datatype/edit-template-export-file.component.html

```diff
@@ -0,0 +1,33 @@
+@if (exportServerEnabled === false) {
+  <mat-form-field>
+    @if (!hideLabel) {
+      <mat-label>
+        <span [matTooltip]="label" matTooltipPosition="before">{{
+          label
+        }}</span>
+      </mat-label>
+    }
+    <input
+      matInput
+      readonly
+      disabled
+      value=""
+      i18n-placeholder="placeholder when export server disabled"
+      placeholder="Export server not available"
+    />
+    <mat-hint class="warning" i18n>
+      This feature is currently not enabled for your system. Please contact your
+      system administrator to enable it.
+    </mat-hint>
+  </mat-form-field>
+} @else if (!exportServerEnabled) {
+  <app-feature-loading></app-feature-loading>
```

Comment:
we could consider showing the whole disabled-info-block here also. Did you try how that looks?

Reply:
thanks, updated now